From 37ee4ae0682e819de98627d49ac428949b2aea33 Mon Sep 17 00:00:00 2001
From: Michael Achenbach <machenbach@chromium.org>
Date: Mon, 24 Oct 2022 09:56:28 +0200
Subject: [PATCH] [infra] Fix trybot names to follow naming conventions

The infra/config update is part of:
https://crrev.com/c/3970938

No-Try: true
Bug: chromium:890222
Change-Id: If0e5960266dc932d5566181785a66da685a4df5a
Reviewed-on: https://chromium-review.googlesource.com/c/v8/v8/+/3973070
Reviewed-by: Liviu Rau <liviurau@google.com>
Commit-Queue: Michael Achenbach <machenbach@chromium.org>
Reviewed-by: Alexander Schulze <alexschulze@chromium.org>
Cr-Commit-Position: refs/heads/main@{#83866}
---
 infra/mb/mb_config.pyl | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/infra/mb/mb_config.pyl b/infra/mb/mb_config.pyl
index 17b867649d..916b1c1127 100644
--- a/infra/mb/mb_config.pyl
+++ b/infra/mb/mb_config.pyl
@@ -255,7 +255,7 @@
       'v8_linux64_verify_csa_rel_ng': 'release_x64_verify_csa',
       'v8_linux64_asan_rel_ng': 'release_x64_asan_minimal_symbols',
       'v8_linux64_cfi_rel_ng': 'release_x64_cfi',
-      'v8_linux64_fuzzilli_ng': 'release_x64_fuzzilli',
+      'v8_linux64_fuzzilli_rel_ng': 'release_x64_fuzzilli',
       'v8_linux64_loong64_rel_ng': 'release_simulate_loong64',
       'v8_linux64_msan_rel_ng': 'release_simulate_arm64_msan_minimal_symbols',
       'v8_linux_riscv32_rel_ng': 'release_simulate_riscv32',
@@ -305,9 +305,9 @@
       'v8_linux_arm64_dbg_ng': 'debug_simulate_arm64',
       'v8_linux_arm64_gc_stress_dbg_ng': 'debug_simulate_arm64',
       'v8_linux_mips64el_compile_rel': 'release_simulate_mips64el',
-      'v8_numfuzz_ng': 'release_x64',
+      'v8_numfuzz_rel_ng': 'release_x64',
       'v8_numfuzz_dbg_ng': 'debug_x64',
-      'v8_numfuzz_tsan_ng': 'release_x64_tsan',
+      'v8_numfuzz_tsan_rel_ng': 'release_x64_tsan',
     },
   },
 
-- 
2.35.1

